i18n/uk.toml

```diff
@@ -18,64 +18,73 @@ other = "у"
 
 # Used in sentences such as "All Tags"
 [ui_all]
-other = "все"
+other = "всі"
 
 # Footer text
 [footer_all_rights_reserved]
-other = "Усі права захищені"
+other = "Усі права захищено"
 
 [footer_privacy_policy]
 other = "Політика конфіденційності"
 
-
 # Post (blog, articles etc.)
 [post_byline_by]
 other = "Автор"
 [post_created]
 other = "Створено"
 [post_last_mod]
-other = "Змінено"
+other = "Востаннє змінено"
 [post_edit_this]
-other = "Відредагувати сторінку"
+other = "Редагувати цю сторінку"
 [post_view_this]
 other = "Переглянути джерело сторінки"
 [post_create_child_page]
-other = "Створити додаткову сторінку"
+other = "Створити дочірню сторінку"
 [post_create_issue]
-other = "Запропонувати змінення документації"
+other = "Створити запит щодо документації"
 [post_create_project_issue]
-other = "Запропонувати допрацювання по проекту"
+other = "Створити запит щодо проєкту"
 [post_posts_in]
-other = "Публікації у"
+other = "Публікації в"
 [post_reading_time]
-other = "хвилина читання"
+other = "хв. читання"
```

Comment:
Why so?

Reply:
Thanks for chiming in @idvoretskyi. I've created the following issue to track your question and any follow up discussion and fixes:

- #2271

Reply:
Please answer in the issue @JackArton.